src/CommandHandler.cpp

```diff
@@ -202,8 +182,8 @@ void CommandHandler::handleNew(std::string const &Command) {
   if (MasterPtr) {
```

Comment:
We have discussed this before but I want to re-iterate that I very much dislike implicit conversion of pointers to bool. I very much prefer to do `MasterPtr != nullptr`.